### ISSUE TITLE
Installing wheel fails

### ISSUE THREAD

**Issue by antonagestam** [boxed/pytest-readme#2]
Output from `pip install pytest-readme`:

```
Building wheels for collected packages: pytest-readme
  Building wheel for pytest-readme (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [2 lines of output]
      running bdist_wheel
      error: error in setup.cfg: command 'bdist_wheel' has no such option 'description_file'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pytest-readme
  Running setup.py clean for pytest-readme
Failed to build pytest-readme
Installing collected packages: pytest-readme
  Running setup.py install for pytest-readme ... done
  DEPRECATION: pytest-readme was installed using the legacy 'setup.py install' method, because a wheel could not be built for it. A possible replacement is to fix the wheel build issue reported above. Discussion can be found at https://github.com/pypa/pip/issues/8368
Successfully installed pytest-readme-1.0.0
```

**Comment by boxed** [comment]
...why is pip install building a wheel? This is a very simple package so I'd think the source package would be fine?

**Comment by antonagestam** [comment]
@boxed I have no idea of the pip internals, there's really nothing special about the environment I tried installing this in. Up-to-date versions of Python, pip et.c.

Why does the `[bdist_wheel]` section have `description-file` here? https://github.com/boxed/pytest-readme/blob/master/setup.cfg#L4

That seems to be the issue?

**Comment by boxed** [comment]
I tried doing `pip install pytest-readme` in a modern venv I had open. Worked fine. Strange.

**Comment by antonagestam** [comment]
@boxed Yes, that does seem strange, does that venv have up-to-date versions of pip, setuptools and wheel?

Here's a minimal Dockerfile to reproduce the problem:

```Dockerfile
FROM python:3.10.6-bullseye
ENV PATH="/venv/bin:$PATH"
WORKDIR /app
RUN python -m venv /venv \
 && pip install --upgrade pip setuptools wheel \
 && pip install pytest-readme
```

`docker build --progress=plain .` will show the problematic output.

Again, is there a special reason you want to keep the `description-file` line? That obviously seems to be what's causing the error.

**Comment by boxed** [comment]
No reason, just noticing that I am confused :P

I released a 1.0.1 without that line. Try it out.

**Comment by antonagestam** [comment]
@boxed Gotcha, installing the new version gives no errors, thanks for the very quick fix 🙏
